frontend/www/js/omegaup/components/course/StudentProgress.vue

```diff
@@ -58,38 +66,106 @@
 import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import { omegaup } from '../../omegaup';
 import { types } from '../../api_types';
+import * as ui from '../../ui';
 import T from '../../lang';
+import omegaup_Markdown from '../Markdown.vue';
+import 'v-tooltip/dist/v-tooltip.css';
+import { VTooltip } from 'v-tooltip';
 
-@Component
+@Component({
+  components: {
+    'omegaup-markdown': omegaup_Markdown,
+  },
+  directives: {
+    tooltip: VTooltip,
+  },
+})
 export default class StudentProgress extends Vue {
   @Prop() course!: types.CourseDetails;
   @Prop() student!: types.StudentProgress;
   @Prop() assignments!: omegaup.Assignment[];
 
   T = T;
 
-  score(assignment: omegaup.Assignment): number {
-    if (!this.student.progress.hasOwnProperty(assignment.alias)) {
+  progress(assignmentAlias: string): number {
+    if (!this.student.progress.hasOwnProperty(assignmentAlias)) {
       return 0;
     }
     return (
-      (Object.values(this.student.progress[assignment.alias]).reduce(
+      (Object.values(this.student.progress[assignmentAlias]).reduce(
         (accumulator: number, currentValue: number) =>
           accumulator + currentValue,
         0,
       ) /
-        (Object.values(this.student.progress[assignment.alias]).length * 100)) *
+        (Object.values(this.student.progress[assignmentAlias]).length * 100)) *
       100
     );
   }
 
-  getProblemColor(problemScore: number): String {
+  score(assignmentAlias: string): number {
+    if (!this.student.score.hasOwnProperty(assignmentAlias)) {
+      return 0;
+    }
+    return Math.round(
+      Object.values(this.student.score[assignmentAlias]).reduce(
+        (accumulator: number, currentValue: number) =>
+          accumulator + currentValue,
+        0,
+      ),
+    );
+  }
+
+  points(assignmentAlias: string): number {
+    if (!this.student.points.hasOwnProperty(assignmentAlias)) {
+      return 0;
+    }
+    return Object.values(this.student.points[assignmentAlias]).reduce(
+      (accumulator: number, currentValue: number) => accumulator + currentValue,
+      0,
+    );
+  }
+
+  getProgressDescription(assignmentAlias: string): string {
+    const score = this.score(assignmentAlias);
+    const points = this.points(assignmentAlias);
+    return ui.formatString(T.studentProgressDescription, {
+      score: score,
+      points: points,
+      progress: points != 0 ? ((score / points) * 100).toFixed(2) : 0,
```

Comment:
nit;

```suggestion
      progress: (points != 0 ? ((score / points) * 100) : 0).toFixed(2),
```

así siempre se muestra con dos puntos decimales, en vez de sólo a veces.

Reply:
Listo.